site2/docs/getting-started-standalone.md

```diff
@@ -74,27 +74,19 @@ If you would like to enable those `builtin` connectors, you can download the con
 * using [wget](https://www.gnu.org/software/wget):
 
   ```shell
-  $ wget pulsar:connector_release_url
+  $ wget pulsar:connector_release_url/{connector name}-{{pulsar:version}}.nar
```

Comment:
can you also fix 2.3.0 docs?

Reply:
Yes, we should do it.

Reply:
if 2.2.0 and 2.2.1 version connector_release_url = https://archive.apache.org/dist/pulsar/pulsar-${version}/connectors/ is error.
such as 2.2.0 url like:
https://archive.apache.org/dist/pulsar/pulsar-2.2.0/apache-pulsar-io-connectors-2.2.0-bin.tar.gz

Reply:
@ambition119 This is mainly used for the latest website and future release will re-use this.